packages/data-point/lib/reducer-object/__snapshots__/factory.test.js.snap

```diff
@@ -36,58 +31,43 @@ Array [
           "type": "ReducerFunction",
         },
       ],
-      "typeOf": "TransformExpression",
+      "type": "ReducerList",
     },
   },
   Object {
     "path": Array [
       "b1",
       "b2",
     ],
-    "transform": TransformExpression {
-      "reducers": Array [
-        ReducerObject {
-          "props": Array [
-            Object {
-              "path": Array [
-                "a3",
-              ],
-              "transform": TransformExpression {
-                "reducers": Array [
-                  ReducerEntity {
-                    "asCollection": false,
-                    "entityType": "transform",
-                    "hasEmptyConditional": false,
-                    "id": "transform:entity-name",
-                    "name": "entity-name",
-                    "type": "ReducerEntity",
-                    "withEmptyConditional": false,
-                  },
-                ],
-                "typeOf": "TransformExpression",
-              },
-            },
-            Object {
-              "path": Array [
-                "b3",
-              ],
-              "transform": TransformExpression {
-                "reducers": Array [
-                  ReducerPath {
-                    "asCollection": false,
-                    "body": [Function],
-                    "name": "b1.b2.b3",
-                    "type": "ReducerPath",
-                  },
-                ],
-                "typeOf": "TransformExpression",
-              },
-            },
+    "transform": ReducerObject {
+      "props": Array [
+        Object {
+          "path": Array [
+            "a3",
+          ],
+          "transform": ReducerEntity {
+            "asCollection": false,
+            "entityType": "transform",
+            "hasEmptyConditional": false,
+            "id": "transform:entity-name",
+            "name": "entity-name",
+            "type": "ReducerEntity",
+            "withEmptyConditional": false,
+          },
+        },
+        Object {
+          "path": Array [
+            "b3",
           ],
-          "type": "ReducerObject",
+          "transform": ReducerPath {
+            "asCollection": false,
+            "body": [Function],
+            "name": "b1.b2.b3",
+            "type": "ReducerPath",
+          },
         },
       ],
-      "typeOf": "TransformExpression",
+      "type": "ReducerObject",
```

Comment:
how was this a TransformExpression before and now a ReducerObject?

Reply:
before it was a single ReducerObject being wrapped in a TransformExpression. one improvement in this PR is that we don't unnecessarily wrap single reducers in a ReducerList.